pkgs/tools/networking/minissdpd/default.nix

```diff
@@ -12,7 +12,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libnfnetlink ];
 
-  installFlags = [ "PREFIX=$(out)" "INSTALLPREFIX=$(out)" ];
+  installFlags = [ "DESTDIR=$(out)" "INSTALLPREFIX=''" ];
```

Comment:
See https://github.com/NixOS/nixpkgs/issues/65718

Reply:
I was unaware of that, I'll look into patching the makefile, as it's trying to install an init.d script but appears to be hand coded. I think I can just remove that part as it won't be used on systemd machines.

Reply:
Patched the makefile to install into `$(out)/etc/init.d/` as potentially someone running nix on a different distribution might need/want the init.d script.